react-common/components/controls/Feedback/FeedbackEventListener.ts

```diff
@@ -0,0 +1,125 @@
+import { appId, feedbackFrameUrl } from './configs';
+interface FeedbackRequestEventPayload<T> {
+  Event: string
+  EventArgs: string
+}
+type FeedbackRequestPayloadType = FeedbackRequestEventPayload<any>
```

Comment:
instead of using `any` here, would be nice to instead have a union of types that contain all of the response objects. then the compiler can figure out the typings based on the `event` field in the switch statement you have in `feedbackCallbackEventListener `